Tensile/KernelWriterAssembly.py

```diff
@@ -1539,13 +1539,16 @@ def initKernel(self, kernel, tPA, tPB ):
       self.defineSgpr("PackedSize0", 1)
     if len(kernel["PackedC1IndicesX"]) > 1:
       self.defineSgpr("PackedSize1", 1)
-
-    if kernel["PackSummationDims"]:
-        self.defineSgpr(self.loopCounterName(kernel,self.unrollIdx), 1)
```

Comment:
This case was removed, is it important?

Reply:
yes we should keep this code

Reply:
my mistake. they are now back in latest push